### PR TITLE
Added scripts for download_data and preprocess

### PR DESCRIPTION
**Pull request by AReyH**
Created the scripts folder in order to store the scripts for the project. Added the following scripts:

* preprocessing.py
* download_data.py

### CODE REVIEW

File: scripts/preprocess.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+  for col in less_frequent_cols:
+    keep_instances = df[col].value_counts(normalize=True).reset_index().head(11).index.tolist()
+
+    keep_instances_dict[col] = keep_instances
+
+    df[col] = df[col].apply(lambda x: x if x in keep_instances_dict[col] else 'Other')
+
+
+  df.to_csv(output_filepath)
```
Comment (786213750) [inline scripts/preprocess.py:98]: add index=False flag

File: scripts/preprocess.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+  keep_instances_dict = {}
+
+  for col in less_frequent_cols:
+    keep_instances = df[col].value_counts(normalize=True).reset_index().head(11).index.tolist()
+
+    keep_instances_dict[col] = keep_instances
+
+    df[col] = df[col].apply(lambda x: x if x in keep_instances_dict[col] else 'Other')
+
```
Comment (786213750) [inline scripts/preprocess.py:96]: rename the target from "isalive" to "target"

Comment (786213750) [reply scripts/preprocess.py:96]: also you can change the target 1 to 0 and 0 to 1
`X["target"] = X["target"].map({0: 1, 1: 0})`

File: scripts/preprocess.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+  df.to_csv(output_filepath)
+  #return df
```
Comment (786213750) [inline scripts/preprocess.py:99]: Can we put the train this split in here as well? so your script write two csv (train.csv and test.csv)

**Review by 786213750 — CHANGES_REQUESTED** [review]

File: scripts/preprocess.py (hunk 3 above)
Comment (AReyH) [reply scripts/preprocess.py:99]: For sure!

**Comment by AReyH** [comment]
Added what @786213750 asked for!

Added:

* index=False
* Changed target variable name
* preprocess.py now exports train_df and test_df

**Comment by 786213750** [comment]
> Added what @786213750 asked for!
> 
> Added:
> 
> * index=False
> * Changed target variable name
> * preprocess.py now exports train_df and test_df

Thanks so much! 1 last change put all the generated data csv to "data/processed" folder

**Comment by AReyH** [comment]
Added the processed folder @786213750 .

The change is done mainly with the commands in terminal

**Review by 786213750 — APPROVED** [review]

**Pull request merged**